src/main/webapp/WEB-INF/web.xml

```diff
@@ -103,6 +103,11 @@ http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd"
         <servlet-class>org.jivesoftware.site.DownloadStats</servlet-class>
         <load-on-startup>1</load-on-startup>
     </servlet>
+    <servlet>
+        <servlet-name>DiscourseAPI</servlet-name>
```

Comment:
Does this expose the service to the world?  Is that necessary or can it be just internally used within the app?

Reply:
No, the servlet is not mapped to any endpoint, so it should not be available publicly. On top of that, it only interacts with very limited Discourse API queries - even if it were exposed, the fallout would be minimal.